Makefile

```diff
@@ -37,31 +38,31 @@ compile-pip-dependencies: ## Uses pip-compile to update requirements.txt
 # It is critical that we run pip-compile via the same Python version
 # that we're generating requirements for, otherwise the versions may
 # be resolved differently.
-	docker run -v "$(DIR)/securethenews:/code" -w /code -it python:3.6-slim \
+	docker run -v "$(DIR):/code" -w /code/securethenews -it $(PY_IMAGE) \
 		bash -c 'apt-get update && apt-get install gcc -y && \
-    pip install --require-hashes -r /code/dev-requirements.txt && \
-		pip-compile --generate-hashes --no-header --output-file /code/requirements.txt /code/requirements.in && \
-		pip-compile --generate-hashes --no-header --allow-unsafe --output-file /code/dev-requirements.txt /code/dev-requirements.in'
+		pip install pip-tools && \
```

Comment:
I wonder if it would be a good idea here to pin the version of `pip-tools`? I can also see though waiting to do that until we can also pin the version of `pip`, since it seems like one of the underlying causes of this command failing is an incompatibility between these two libraries.

As a point of reference, the way it's being pinned in one of the other repos is [with a separate requirements file just for pip tools](https://github.com/freedomofpress/tracker/blob/master/Makefile#L49), which also lets us use `--require-hashes` when installing. I don't really love that solution, as it proliferates the number of requirements files and probably ought to have _another_ command to update the pip tools version, and it kind of seems like a lot of commands and files but it does pin the version and require the hash.

It also seems perfectly reasonable to just `pip install pip-tools` like you have here. I don't know.. what do you think?